sections/futures/TradingHistory/TradesHistoryTable.tsx

```diff
@@ -88,29 +89,24 @@ const TradesHistoryTable: FC<TradesHistoryTableProps> = ({ currencyKey, numberOf
 							Header: <TableHeader>{t('futures.market.history.amount-label')}</TableHeader>,
 							accessor: 'Amount',
 							Cell: (cellProps: CellProps<any>) => {
+								const numValue = Math.abs(cellProps.row.original.amount / 1e18);
+								const numDecimals =
+									numValue === 0 ? 2 : numValue < 1 ? 4 : numValue >= 100000 ? 0 : 2;
+
+								const normal = cellProps.row.original.orderType === 'Liquidation';
+								const negative = cellProps.row.original.amount > 0;
+
 								return (
 									<>
-										{cellProps.row.original.amount > 0 ? (
-											<div>
-												<PostiveValue>
-													{cellProps.row.original.amount !== NO_VALUE
-														? formatNumber(Math.abs(cellProps.row.original.amount / 1e18), {
-																minDecimals: 4,
-														  })
-														: NO_VALUE}
-												</PostiveValue>
-											</div>
-										) : (
-											<div>
-												<NegativeValue>
-													{cellProps.row.original.value !== NO_VALUE
-														? formatNumber(Math.abs(cellProps.row.original.amount / 1e18), {
-																minDecimals: 4,
-														  })
-														: NO_VALUE}
-												</NegativeValue>
-											</div>
-										)}
+										<div>
+											<DirectionalValue negative={negative} normal={normal}>
+												{cellProps.row.original.amount !== NO_VALUE
+													? `${formatNumber(numValue, {
+															minDecimals: numDecimals,
+													  })} ${normal ? '💀' : ''}`
```

Comment:
ha nice - is this suppose to stay?

Reply:
Yeah, we will leave it here but removed the one from the "Trades" tab